README.md

```diff
@@ -15,6 +15,12 @@ SOME_OTHER_CONFIG=something
 3. Rename your new project using react-native-rename (https://github.com/junedomingo/react-native-rename)
 4. Start on android or ios: `react-native run-android` or `react-native run-ios`
 
+## Build Android Release
+1. Ask a developer for the release key and place it in `/android/app`
```

Comment:
There's a folder keystores in the project, maybe is more suited to put the keystore there.